src/components/AppCatalog/AppDetail/ClusterPicker.tsx

```diff
@@ -26,16 +27,27 @@ const Cluster = styled.div`
   &.selected {
     background-color: ${(props) => props.theme.colors.shade4};
   }
+  &.disabled {
+    opacity: 0.5;
```

Comment:
There's the `disabledOpacity` [property in the theme](https://github.com/giantswarm/happa/blob/f23b6e9873d2ae8e4dc5de8eea38c462c4e5351d/src/styles/theme.ts#L78). We can maybe adjust that, if it doesn't look *disabled* enough